test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithAllFeatures.cs

```diff
@@ -147,6 +147,97 @@ public static IEnumerable<object[]> PublishData
                         "NoneCopyOutputAlways.txt",
                         "NoneCopyOutputPreserveNewest.txt",
                         "CopyToOutputFromProjectReference.txt",
+                        "System.Spatial.dll",
+                        "System.AppContext.dll",
```

Comment:
No, it's not really good- one of the motivations for targeting .NET Core 1.1 by default is that things like this happen when targeting 1.0.

In this case, those files are getting published because the app is referencing a library, and the library is using version 1.6.1 of the NETStandard.Library package.  We originally tried to fix this by using version 1.6.0 of NETStandard.Library instead (#793), but that leads to downgrade issues and still doesn't fix the problem if the app ends up transitively depending on System.ValueTuple, the latest version of immutable collections, or an ever-growing number of other packages.

If you are targeting .NET Core 1.0 and aren't referencing any NuGet packages that would bring in assemblies from the 1.1 band, then you can avoid getting local copies of these assemblies by setting `NetStandardImplicitPackageVersion` to `1.6.0` in all of your library projects.

But really, I'd recommend you just target .NET Core 1.1 instead.